content/pages/info/diversity/ca.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Not sure if we should still carry the Spanish/Catalan versions of these? Maybe a Dutch version instead?

Reply:
Yeah, definitely drop the spanish/catalan versions.
If you want we can add a dutch version, but it would also be fine to add that later.